lib/Sema/ConstraintSystem.cpp

```diff
@@ -928,6 +928,82 @@ TypeVariableType *ConstraintSystem::isRepresentativeFor(
   return *member;
 }
 
+static Optional<std::pair<VarDecl *, Type>>
+getPropertyWrappersInformationFromOverload(
+    SelectedOverload resolvedOverload, DeclContext *DC,
+    llvm::function_ref<Optional<std::pair<VarDecl *, Type>>(VarDecl *)>
+        getInformation) {
+  if (resolvedOverload.choice.isDecl()) {
+    if (auto *decl = dyn_cast<VarDecl>(resolvedOverload.choice.getDecl())) {
```

Comment:
Nit: there is `getDeclOrNull()` method on `OverloadChoice` so checking for `isDecl()` is unnecessary.

Reply:
Fixed :)